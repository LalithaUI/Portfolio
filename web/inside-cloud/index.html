<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
<style>
    body {
        background: rgb(38, 198, 247);
    }
    div {
        position: absolute;
        top: 50%;
        left: 50%;
        transform: translate(-50%, -50%);
        color: #fff;
        text-align: center;
        font-family: Verdana, Geneva, Tahoma, sans-serif;
    }
</style>
</head>
<body>
    <div>
        <h1>Inside Cloud</h1>
        <p>This page under developing please try another page to open</p>
    </div>
</body>
</html>