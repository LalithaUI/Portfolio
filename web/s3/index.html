<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">
  <link rel="canonical" href="http://s3techgrp.com/" />
  <title>S3 Technology Group</title>
  <meta content="S3 Technology Group is a leading IT Service product company offering innovative services with global delivery excellence. If you are looking to improve the efficiency of the company and multiply your profits, we are right here with our professionals to assist you" name="description">
  <meta content="S3,S3 Technology,S3 Technology Group,S3 Group,Data analystics,Data Science,Artificial Intelligence,Machine Learning,Contractual Outsourcing,Cloud Computing,Network Engineers,IT Consulting,DevOps Engineers,Salesforce Developers,Java Developers,ServiceNow Developers" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.ico" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
 
  <link href="assets/vendor/icofont/icofont.min.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/animate.css/animate.min.css" rel="stylesheet">
  <link href="assets/vendor/remixicon/remixicon.css" rel="stylesheet">
  <link href="assets/vendor/venobox/venobox.css" rel="stylesheet">
  <link href="assets/vendor/owl.carousel/assets/owl.carousel.min.css" rel="stylesheet">
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  
</head>

<body>

  <!-- ======= Header ======= -->
  <header id="header" class="fixed-top">
    <div class="container d-flex align-items-center">

     
      <!-- Uncomment below if you prefer to use an image logo -->
      <a href="index.html" class="logo mr-auto"><img src="assets/img/logo2.png" alt="" class="img-fluid"></a>

      <nav class="nav-menu d-none d-lg-block">
        <ul>
          <li class="active"><a href="index.html">Home</a></li>
          <li><a href="about.html">About Us</a></li>
          <li><a href="services.html">Services</a></li>
          <li><a href="contact.html">Contact Us</a></li>

        </ul>
      </nav><!-- .nav-menu -->

      

    </div>
  </header><!-- End Header -->

  <!-- ======= Hero Section ======= -->
  <section id="hero">
    <div id="heroCarousel" class="carousel slide carousel-fade" data-ride="carousel">

      <ol class="carousel-indicators" id="hero-carousel-indicators"></ol>

      <div class="carousel-inner" role="listbox">

        <!-- Slide 1 -->
        <div class="carousel-item active" style="background-image: url(assets/img/slide/businessmen-working-strategic-planning1.jpg)">
          <div class="carousel-container">
            <div class="container">
              <h2 class="animate__animated animate__fadeInDown">Welcome to <span>S3</span></h2>
              <p class="animate__animated animate__fadeInUp">S3 Technology Group is a leading IT Service product company offering innovative services with global delivery excellence. If you are looking to improve the efficiency of the company and multiply your profits, we are right here with our professionals to assist you.
              </p>
              
            </div>
          </div>
        </div>

        <!-- Slide 2 -->
        <div class="carousel-item" style="background-image: url(assets/img/slide/business-handshake-technology-corporate-business-concept.jpg)">
          <div class="carousel-container">
            <div class="container">
              <h2 class="animate__animated animate__fadeInDown">Whats our hirings apart from others?</h2>
              <p class="animate__animated animate__fadeInUp">S3 Technology Group works as a one-stop solution for the business and employees. We support our clients to find potential and knowledgeable professionals. Similarly,  Professionals get an opportunity to discover their dream organisation with multiple availability suiting their needs.</p>
              
            </div>
          </div>
        </div>

        <!-- Slide 3 -->
        <div class="carousel-item" style="background-image: url(assets/img/slide/slide-3.jpg)">
          <div class="carousel-container">
            <div class="container">
              <h2 class="animate__animated animate__fadeInDown">Uniqueness that makes you choose us!</h2>
              <p class="animate__animated animate__fadeInUp">Vast spectrum of satisfied clients reveals the diligence we put in each and every requirement. S3 Technology Group outsource service reduces the expenditure, contributes to business development, and eliminates the hectic need of hiring, deterring, and selecting the candidates.</p>
              
            </div>
          </div>
        </div>

      </div>

      <a class="carousel-control-prev" href="#heroCarousel" role="button" data-slide="prev">
        <span class="carousel-control-prev-icon icofont-simple-left" aria-hidden="true"></span>
        <span class="sr-only">Previous</span>
      </a>

      <a class="carousel-control-next" href="#heroCarousel" role="button" data-slide="next">
        <span class="carousel-control-next-icon icofont-simple-right" aria-hidden="true"></span>
        <span class="sr-only">Next</span>
      </a>

    </div>
  </section><!-- End Hero -->

  <main id="main">

    <!-- ======= Mission Section ======= -->
    <section  class="about">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>S3</h2>
          <p>S3 Technology Group</p>
        </div>

        <div class="row content">
          <div class="col-lg-6">
           <img src="assets/img/section/mission.jpg" width="95%" class="anim" style="z-index:-1;">
          </div>
          <div class="col-lg-6 pt-4 pt-lg-0">
            <h4 style="margin-bottom:20px;">Our Quick Introduction as a Leading Consulting Company</h4>
            <p>
              S3 Technology Group is a leading IT Service product company offering innovative services with global delivery excellence. If you are looking to improve the efficiency of the company and multiply your profits, we are right here with our professionals to assist you.
            </p>
           
            <p>
              Our company plays a prominent role in the success of a business. We are known for assessing the right needs of the business professionals and providing the same with our exclusive and technology-integrated training. 
            </p>
            
          </div>
        </div>

      </div>
    </section><!-- End Mission Section -->




   
    <!-- ======= Services Section ======= -->
    <section  class="services">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Services</h2>
          <p>Check our Services</p>
        </div>

        <div class="row">
          <div class="col-lg-3 col-md-6 d-flex align-items-stretch" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/data-scientist.png" width="80%"></div>
              <h4><a href="">Data analystics/Data Sceince</a></h4>
              <p>Our experts have state-of-the-art experience in availing the trends <a href="services.html">Read more...</a></p>
            </div>
          </div>

          

          

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/artificial-intelligence.png" width="80%"></div>
              <h4><a href="">Artificial Intelligence</a></h4>
              <p>Our trained and qualified professionals utilize cutting edge methods <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/machine-learning.png" width="80%"></div>
              <h4><a href="">Machine Learning</a></h4>
              <p>Our highly experienced professionals have a good grasp over utilizing <a href="services.html">Read more...</a> </p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/networking.png" width="80%"></div>
              <h4><a href="">Contractual Outsourcing</a></h4>
              <p>We work towards your organisations development and at the same<a href="services.html"> Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/big-data.png" width="80%"></div>
              <h4><a href="">Cloud Computing</a></h4>
              <p>We have trained cloud engineers who have in-depth knowledge of database query <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/structure.png" width="80%"></div>
              <h4><a href="">Network Engineers</a></h4>
              <p>Our engineers are committed to configure and install various network <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/consultation.png" width="80%"></div>
              <h4><a href="">IT Consulting</a></h4>
              <p>We provide expert analysts to evaluate the IT systems of your organisation <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/seo.png" width="80%"></div>
              <h4><a href="">DevOps Engineers</a></h4>
              <p>Well experienced and knowledgeable professionals are always at  <a href="services.html">Read more...</a></p>
            </div>
          </div>
          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/default copy.png" width="80%"></div>
              <h4><a href="">Salesforce Developers</a></h4>
              <p>Our knowledgeable team is eager to develop business workflows for your  <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/java.png" width="80%"></div>
              <h4><a href="">Java Developers</a></h4>
              <p>Our highly experienced and qualified programmers will design and <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/images.png" width="80%"></div>
              <h4><a href="">.net/Microsoft Suite Developers</a></h4>
              <p>Our potential team is always on the toes to create the application from <a href="services.html">Read more...</a></p>
            </div>
          </div>

          <div class="col-lg-3 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box">
              <div class="icon"><img src="assets/img/icon/servicenow copy.png" width="80%"></div>
              <h4><a href="">ServiceNow Developers</a></h4>
              <p>Our specialized team is committed to design and develop technical<a href="services.html"> Read more...</a></p>
            </div>
          </div>

        </div>

      </div>
    </section><!-- End Services Section -->

      
    <!-- ======= Approach Section ======= -->
    <section  class="about">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Approach</h2>
          <p>Our Approach</p>
        </div>

        <div class="row content">
          
          <div class="col-lg-6 pt-4 pt-lg-0">
            <img src="assets/img/section/approach.jpg" width="95%" class="anim" style="z-index:-1;">
          </div>

          <div class="col-lg-6">
            <p>
              S3 Technology Group works as a one-stop solution for the business and employees. We support our clients to find potential and knowledgeable professionals. Similarly,  Professionals get an opportunity to discover their dream organisation with multiple availability suiting their needs. 
            </p>
            
            <p>
              We study customers' requirements and needs through in-depth knowledge of the IT technical field. Our team of professionals determine and list out the needs of the clients and filter the employees. This way technology-integrated training is offered so that they can strive for growth and profitability.

            </p>
            <p>Vast spectrum of satisfied clients reveals the diligence we put in each and every requirement. S3 Technology Group outsource service reduces the expenditure, contributes to business development, and eliminates the hectic need of hiring, deterring, and selecting the candidates.</p>
            
          </div>
        </div>

      </div>
    </section><!-- End Approach Section -->

    

    
    
    

  </main><!-- End #main -->

  <!-- ======= Footer ======= -->
  <footer id="footer">
    <div class="footer-top">
      <div class="container">
        <div class="row">

          <div class="col-lg-6 col-md-6">
            <div class="footer-info">
              <h3>S3 Technology Group</h3>
              <p class="pb-3"><em>Looking for the opportunity to grab the best employees and improve the efficiency and profitability of the company, reach out to S3 Technology Group.</em></p>
              <p>
                
                <strong>Contact:</strong> 209-666-0057<br>
                <strong>Office:</strong> 209-299-3223<br>
                <strong>Email:</strong> info@s3techgrp.com<br>
              </p>
              <div class="social-links mt-3">
                <a href="#" class="twitter"><i class="bx bxl-twitter"></i></a>
                <a href="#" class="facebook"><i class="bx bxl-facebook"></i></a>
                <a href="#" class="instagram"><i class="bx bxl-instagram"></i></a>
                <a href="#" class="google-plus"><i class="bx bxl-skype"></i></a>
                <a href="#" class="linkedin"><i class="bx bxl-linkedin"></i></a>
              </div>
            </div>
          </div>

          <div class="col-lg-3 col-md-3 footer-links">
            <h4>Useful Links</h4>
            <ul>
              <li><i class="bx bx-chevron-right"></i> <a href="index.html">Home</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="about.html">About us</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Services</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="contact.html">Contact Us</a></li>
              
            </ul>
          </div>

          <div class="col-lg-3 col-md-3 footer-links">
            <h4>Our Services</h4>
            <ul>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Data Analytics/Data Science</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Artificial Intelligence</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Machine Learning</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Contractual Outsourcing</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Cloud Computing</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="services.html">Network Engineers</a></li>
              
            </ul>
          </div>

         

        </div>
      </div>
    </div>

    <div class="container">
      <div class="copyright">
        &copy; Copyright <strong><span>S3 Technology Group</span></strong>. All Rights Reserved
      </div>
      
    </div>
  </footer><!-- End Footer -->

  <div id="preloader" class="text-center" style="margin:auto;"><img src="assets/img/logo.png" alt="" class="img-fluid reloader-img"></div>
  <a href="#" class="back-to-top"><i class="icofont-simple-up"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/jquery/jquery.min.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  
  <script src="assets/vendor/jquery.easing/jquery.easing.min.js"></script>
  <script src="assets/vendor/waypoints/jquery.waypoints.min.js"></script>
  <script src="assets/vendor/counterup/counterup.min.js"></script>
  <script src="assets/vendor/owl.carousel/owl.carousel.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/aos/aos.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>