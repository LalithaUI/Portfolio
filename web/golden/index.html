<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Vrhunec</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
    <link rel="stylesheet" href="style.css">
</head>
<body>
   
    <header>
            <div class="container">
                <div class="main">
                    <div class="logo">
                        <a href="#">golden</a>
                    </div>
                    <div class="navigation">
                       
                        <input type="checkbox" class="toggle-menu">
                        <div class="hamburger"></div>
                        <ul class="menu">
                            <li>
                                <a href="#home">Home</a>
                            </li>
                            <li>
                                <a href="#services">Services</a>
                            </li>
                            <li>
                                <a href="#portfolio">Portfolio</a>
                            </li>
    
                            <li>
                                <a href="#about">About</a>
                            </li>
                            <li>
                                <a href="#contact">Contact</a>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
    </header>
  
    <div class="banner">
        <div class="container">
            <div class="main-banner">
                <p>Welcome To Our Studio!</p>
                <p>it's nice to meet you</p>
                <a href="#tellMore">Tell me more</a>
            </div>
        </div>
    </div>
   
    <section class="services">
        <div class="container">
            <h1>services</h1>
            <p>Proin iaculis purus consequat sem cure. </p>
            <div class="services-main">
                <div class="services-block">
                    <div class="img-circle">
                        <img src="img/icon-basket.png" alt="icon-basket">
                    </div>
                    <h3>E-Commerce</h3>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
                <div class="services-block">
                    <div class="img-circle">
                        <img src="img/icon-laptop.png" alt="icon-basket">
                    </div>
                    <h3>Responsive Web</h3>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
                <div class="services-block">
                    <div class="img-circle">
                        <img src="img/icon-locked.png" alt="icon-basket">
                    </div>
                    <h3>Web Security</h3>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
            </div>
        </div>
    </section>
    
    <section class="contact"> 
            <div class="container">
                <h1>contact us</h1>
                <p>Proin iaculis purus consequat sem cure. </p>
                <form action="#">
                    <div class="contact-main">
                        <div class="form-left">
                            <input type="text" placeholder="your name*" name="name">
                            <input type="email" placeholder="your email*" name="email">
                            <input type="text" placeholder="subject" name="subject">
                        </div>
                        <div class="form-right">
                            <textarea name="comment" id="comment" cols="30" rows="10" placeholder="your message*"></textarea>
                        </div>  
                    </div>
                    <input type="submit" name="submit" id="submit" value="SEND MESSAGE" class="submit">
                </form>
            </div>
    </section>
    
    <footer>
        <div class="container">
            <div class="footer-main">
                <div class="footer-copy-right">
                    <p>&#169; Copyright 2014 FreebiesXpress.com</p>
                </div>
                <div class="social-links">
                    <ul>
                        <li>
                            <a href="#twitter"><i class="fa-brands fa-twitter"></i></a>
                        </li>
                        <li>
                            <a href="#facebook"><i class="fa-brands fa-facebook-f"></i></a>
                        </li>
                        <li>
                            <a href="#pinterest"><i class="fa-brands fa-pinterest-p"></i></a>
                        </li>
                        <li>
                            <a href="#google-plus">
                                <img src="img/google-plus.png" alt="google-plus">
                            </a>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </footer>
</body>
</html>